msi/release.sh

```diff
@@ -206,12 +206,26 @@ EOF
 mkdir -p obj &&
 wix/candle.exe -dVersion="${VERSION%%-*}" \
 	-arch $WIX_ARCH \
-    GitProduct.wxs GitComponents.wxs -o obj\\ \
-	-ext WixUtilExtension -ext WixTagExtension &&
+    GitProduct.wxs GitComponents.wxs \
+    GitChooseEnvironmentDlg.wxs GitChooseTerminalDlg.wxs \
+    GitCustomizeLineEndingsDlg.wxs GitLineEndingsDlg.wxs \
+    GitMaintenanceTypeDlg.wxs GitPerformanceOptionsDlg.wxs \
+    -o obj\\ \
+    -ext WixUtilExtension -ext WixTagExtension \
+    -ext WixUIExtension &&
 wix/light.exe \
 	obj/GitProduct.wixobj \
     obj/GitComponents.wixobj \
+	obj/GitChooseEnvironmentDlg.wixobj \
+	obj/GitChooseTerminalDlg.wixobj \
+	obj/GitCustomizeLineEndingsDlg.wixobj \
+	obj/GitLineEndingsDlg.wixobj \
+	obj/GitMaintenanceTypeDlg.wixobj \
+	obj/GitPerformanceOptionsDlg.wixobj \
 	-o $TARGET -ext WixUtilExtension -ext WixTagExtension \
+	-ext WixUIExtension -loc GitUI_en-us.wxl \
+	-cultures:en-us \
+	-spdb \
```

Comment:
@robmen I did this because of [this issue](https://github.com/wixtoolset/issues/issues/5169) over on the WiX Toolset issues list. Otherwise, _completely_ I agree with you.

I'll try to re-enable it and see if I run into this issue. I'm thinking this _might_ have something do with command line arguments appearing after the `-b` parameters--but I haven't been able to confirm this yet.